src/screens/Login.js

```diff
@@ -118,39 +134,65 @@ class Login extends Component {
   render() {
     const { error } = this.state;
     return (
-      <KeyboardAvoidingView behavior="padding" style={styles.container}>
-        <View style={styles.backgroundStyle}>
-          <View style={styles.viewStyle}>
-            <OpacityButton text="Play without logging!" onPress={this.onAnonymous} />
-            <Text style={styles.signUpText}>
-              Sign up/in
-            </Text>
-            <Text style={styles.infoText}>
-              After logging you would see history of your plays
-            </Text>
-            <StatusBar barStyle="light-content" />
+      <ImageBackground
+        style={styles.backgroundImage}
+        source={require('./../../assets/background.png')}
+      >
+        <KeyboardAvoidingView behavior="padding" style={[styles.view, styles.keyboardAvoidingView]}>
```

Comment:
I like that we use keyboard avoiding here but right now it covers hides this `No account yet? sign up!` text. That would be awesome to maybe hide the question mark logo while typing in login and password? What do you gusy think? @chudy1997 @ign699 @Baksztag